# /database/srv/SetPotInfo.srv

PotInfo info
---
bool success